packages/math-input/src/components/keypad/index.tsx

```diff
@@ -3,25 +3,31 @@ import * as React from "react";
 
 import Tabbar from "../tabbar/tabbar";
 
-import NumericInputPage from "./numeric-input-page";
-import PreAlgebraPage from "./pre-algebra-page";
-import TrigonometryPage from "./trigonometry-page";
+import TrigonometryPage from "./geometry-page/trigonometry-page";
+import NumericInputPage from "./numbers-page/numeric-input-page";
+import OperatorsPage, {
+    OperatorsButtonSets,
+} from "./operators-page/operators-page";
```

Comment:
I _think_ this would be cleaner if we created a file `operations-page/index.tsx` and then this import could just be `from "./operations-page`.  

That index.ts would export both the component from `operations-page.tsx` and the `OperatorsButtonSets`. 
```suggestion
import OperatorsPage, { OperatorsButtonSets } from "./operators-page";
```

Reply:
Woohoo! Already doing this on my working branch.